### DISCUSSION TITLE
SUGGESTION: Cromha-Expansion plugin website

### DISCUSSION THREAD

**Discussion by lumbar527** [OcelotWalrus/Cromha-Expansion-plugin#131] [Plugin Enhancment]
I would like to make a website for the Cromha-Expansion plugin. I have already made some files but they are only on my computer. 
<details><summary>Screenshots</summary>
<p>
Homepage:
<img width="841" alt="Screen Shot 2023-09-17 at 12 45 13 PM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/a2ec68ac-beac-4066-afad-29fc8d8c08f3">
Gallery Page:
<img width="858" alt="Screen Shot 2023-09-18 at 11 57 26 AM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/638fe8c2-a14f-474d-add5-ec69b78f1f40">
<img width="667" alt="Screen Shot 2023-09-18 at 11 57 41 AM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/06364236-3afa-46ee-bf35-c3ce44757e5f">
<img width="689" alt="Screen Shot 2023-09-18 at 11 57 58 AM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/192dace8-d0f4-4d64-bf85-39b165eaa9aa">
<img width="656" alt="Screen Shot 2023-09-18 at 11 58 08 AM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/eb8528f7-e7be-4949-8fc5-a4471e06d029">

</p>
</details>

**Comment by lumbar527** [comment]
I'm currently working on the potential website.

**Comment by lumbar527** [comment]
Here's a potential website design, based of off the Dungeons-of-Kathallion website (I don't know that much html/css stuff):
<img width="1682" alt="Screen Shot 2023-09-17 at 12 45 13 PM" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/01e2a8cb-8566-44de-9d11-a684f5ed479c">

**Comment by OcelotWalrus** [comment]
I don't really know if making a website would be really needed or useful. Because still it's an idea.

**Reply by lumbar527** [reply]
OK, I'll put this down for now.
